sdk/cloudmachine/Azure.CloudMachine/api/Azure.CloudMachine.netstandard2.0.cs

```diff
@@ -0,0 +1,43 @@
+namespace Azure.Template
```

Comment:
I don't think we want this, do we?

Reply:
Forgot to re-export. It should be fixed now